cmd/yggd/grpc.go

```diff
@@ -16,6 +17,10 @@ import (
 	"google.golang.org/grpc"
 )
 
+var (
+	sendResponseTimeout = 20 * time.Second
```

Comment:
Is this longer than the gRPC connection timeout? Or does that default to 30 seconds.

Reply:
Ok, I try to get to what server is defined, by default is 120 seconds a bit excessive IMHO. 

The setting is  from here: 
https://pkg.go.dev/google.golang.org/grpc#NewServer

And the value is inherit from here: 
https://github.com/grpc/grpc-go/blob/v1.44.0/server.go#L173

So, I'm happy to do some kind of limit to 20-30  on the server side or if you want move this to 120seconds. 

Open here.